src/org/ods/OdsPipeline.groovy

```diff
@@ -91,7 +103,26 @@ class OdsPipeline implements Serializable {
     logger.info "***** Finished ODS Pipeline *****"
   }
 
+  def checkForMultiRepoBuild() {
+    if (!!script.env.MULTI_REPO_BUILD) {
+      logger.info '***** Multi Repo Build detected *****'
+      context.bitbucketNotificationEnabled = false
+      context.localCheckoutEnabled = false
+      context.displayNameUpdateEnabled = false
+      context.ciSkipEnabled = false
+      context.mailNotificationEnabled = false
+      def buildEnv = script.env.MULTI_REPO_ENV
+      if (buildEnv) {
+        context.environment = buildEnv
+        context.cloneSourceEnv = null
+      }
```

Comment:
Let's handle the unexpected case that the env variable `MULTI_REPO_ENV` is not present.